setup.cfg

```diff
@@ -20,16 +20,16 @@ classifiers =
     Intended Audience :: Science/Research
     License :: OSI Approved :: BSD License
     Programming Language :: Python
-    Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
 
 [options]
 zip_safe = False
 include_package_data = True
 packages = find:
-python_requires = >=3.6
+python_requires = >=3.7
```

Comment:
Do we really want to forbid installing JupyterLab on older system? Or just mentioned through the specifiers it is not supported.

Forbidding it may be too stringent on some servers that ships with older stable OS. More over there is no code used that requires Python >= 3.7

Reply:
Maybe we can also wait for `jupyter_server` (and / or other dependencies) to drop it first.

Reply:
Based on the discussions from yesterday's meeting (https://github.com/jupyterlab/team-compass/issues/128#issuecomment-995035053), I pushed https://github.com/jupyterlab/jupyterlab/pull/11646/commits/088a672075ae3cd837d86fa500c4e78db24e0106 to keep `python_requires = >=3.6` for now.

So we can move forward with this PR and do the real drop at a later stage.